src/main/java/edu/kis/powp/jobs2d/features/DrawerPanelClickMouseListenerFeature.java

```diff
@@ -0,0 +1,61 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+
+public class DrawerPanelClickMouseListenerFeature implements MouseListener, Job2dDriver {
+    private ILine line;
+    private int startX = 0, startY = 0;
+    private DrawPanelController drawController;
+
+    public DrawerPanelClickMouseListenerFeature(DrawPanelController drawController, ILine line) {
+        super();
+        this.drawController = drawController;
+        this.line = line;
+    }
+
+    @Override
+    public void mouseClicked(MouseEvent mouseEvent) {
+
+    }
+
+    @Override
+    public void mousePressed(MouseEvent mouseEvent) {
+        startX = mouseEvent.getX()-268;
+        startY = mouseEvent.getY()-228;
+        setPosition(startX, startY);
```

Comment:
The goal is to call a current app driver (manager.getCurrentDriver), not draw directly.